### PR TITLE
Confirming a booking removes availability

### LINKED ISSUE #8: 9 - Double booked spaces
``` 
As a user 
So my spaces dont get double booked
I want availability to be removed upon booking
```

### LINKED ISSUE #9: 8 - removal of availability
```
As an owner
So that I can review multiple requests 
I do not want availability to be removed until I have confirmed a booking
```

### PR DESCRIPTION
**Pull request by alextwilson**
Closes #9 
Closes #8 

Page for a users spaces and requests
Button that checks if request is still vaild then removes availability
Rendering on page changes to show that booking is confirmed

### CODE REVIEW

File: app/app.rb [hunk 1]
```diff
@@ -44,7 +44,8 @@ class DwellBNB < Sinatra::Base
   post '/spaces' do
     space = Space.new(name: params[:name],
                 description: params[:description],
-                price: params[:price], picture_url: params[:picture_url])
+                price: params[:price], picture_url: params[:picture_url],
+                user_id: current_user.id)
```
Comment (Leigan0) [inline app/app.rb:48]: might need a flash error or HTML error or the spaces wont save if not logged in

Comment (alextwilson) [reply app/app.rb:48]: Sorted

**Comment by Leigan0** [comment]
👍

**Review by ghost — COMMENTED** [review]
👍

**Comment by Gleoman** [comment]
:shipit:

**Pull request merged**